### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by Alexander-Wilms**

### CODE REVIEW

**Comment by Alexander-Wilms** [comment]
I went through all the changes, so it should be mergeable without someone reviewing everything again.

File: lib/CGameInfoCallback.cpp [hunk 1]
```diff
@@ -781,12 +781,12 @@ int CPlayerSpecificInfoCallback::getHeroSerial(const CGHeroInstance * hero, bool
...
 
-	for (auto & heroe : heroes)
+	for (auto & hero : heroes)
 	{
-		if (includeGarrisoned || !(heroe)->inTownGarrison)
+		if (includeGarrisoned || !(hero)->inTownGarrison)
 			index++;
 
-		if (heroe == hero)
+		if (hero == hero)
```
Comment (IvanSavenko) [inline lib/CGameInfoCallback.cpp:789]: Something went wrong here.

File: lib/CSoundBase.h [hunk 2]
```diff
@@ -259,7 +259,7 @@ VCMI_SOUND_NAME(Dig) VCMI_SOUND_FILE(DIGSOUND.wav) \
 VCMI_SOUND_NAME(DIPMAGK) VCMI_SOUND_FILE(DIPMAGK.wav) \
 VCMI_SOUND_NAME(DISEASE) VCMI_SOUND_FILE(DISEASE.wav) \
 VCMI_SOUND_NAME(DISGUISE) VCMI_SOUND_FILE(DISGUISE.wav) \
-VCMI_SOUND_NAME(DISPELL) VCMI_SOUND_FILE(DISPELL.wav) \
+VCMI_SOUND_NAME(DISPEL) VCMI_SOUND_FILE(DISPEL.wav) \
```
Comment (dydzio0614) [inline lib/CSoundBase.h:262]: @IvanSavenko should file name change stay as it was?

File: lib/CSoundBase.h [hunk 3]
```diff
@@ -842,7 +842,7 @@ VCMI_SOUND_NAME(RDDRMove) VCMI_SOUND_FILE(RDDRMOVE.wav) \
 VCMI_SOUND_NAME(RDDRWNCE) VCMI_SOUND_FILE(RDDRWNCE.wav) \
 VCMI_SOUND_NAME(REGENER) VCMI_SOUND_FILE(REGENER.wav) \
 VCMI_SOUND_NAME(REMoveOB) VCMI_SOUND_FILE(REMOVEOB.wav) \
-VCMI_SOUND_NAME(RESURECT) VCMI_SOUND_FILE(RESURECT.wav) \
+VCMI_SOUND_NAME(RESURRECT) VCMI_SOUND_FILE(RESURRECT.wav) \
```
Comment (dydzio0614) [inline lib/CSoundBase.h:845]: file name changed here as well

File: mapeditor/mapsettings/eventsettings.cpp [hunk 4]
```diff
@@ -41,8 +41,8 @@ QVariant toVariant(const CMapEvent & event)
 	result["players"] = QVariant::fromValue(event.players);
 	result["humanAffected"] = QVariant::fromValue(event.humanAffected);
 	result["computerAffected"] = QVariant::fromValue(event.computerAffected);
-	result["firstOccurence"] = QVariant::fromValue(event.firstOccurence);
-	result["nextOccurence"] = QVariant::fromValue(event.nextOccurence);
+	result["firstOccurrence"] = QVariant::fromValue(event.firstOccurrence);
```
Comment (dydzio0614) [inline mapeditor/mapsettings/eventsettings.cpp:44]: does this have chance to break compatibility with existing maps?

File: launcher/modManager/cmodmanager.cpp [hunk 5]
```diff
@@ -112,7 +112,7 @@ void CModManager::loadMods()
 			auto json = JsonUtils::toJson(mod);
 			json["localSizeBytes"].Float() = total;
 			if(!name.is_absolute())
-				json["storedLocaly"].Bool() = true;
+				json["storedLocally"].Bool() = true;
```
Comment (Alexander-Wilms) [inline launcher/modManager/cmodmanager.cpp:115]: Would this cause issues with existing JSON files?

File: lib/CSoundBase.h (hunk 2 above)
Comment (Alexander-Wilms) [reply lib/CSoundBase.h:262]: Does the change to `VCMI_SOUND_NAME` have to be reverted as well?

File: mapeditor/inspector/inspector.cpp [hunk 6]
```diff
@@ -464,7 +464,7 @@ void Inspector::updateProperties()
 		return;
 	table->setRowCount(0); //cleanup table
 	
-	addProperty("Indentifier", obj);
+	addProperty("Identifier", obj);
```
Comment (Alexander-Wilms) [inline mapeditor/inspector/inspector.cpp:467]: Would this be problematic?

Comment (IvanSavenko) [reply mapeditor/inspector/inspector.cpp:467]: This should be safe. Seems to be only used inside of map editor

File: launcher/modManager/cmodmanager.cpp (hunk 5 above)
Comment (IvanSavenko) [reply launcher/modManager/cmodmanager.cpp:115]: Safe. Unused code actually. Property is never read.

File: mapeditor/mapsettings/eventsettings.cpp (hunk 4 above)
Comment (IvanSavenko) [reply mapeditor/mapsettings/eventsettings.cpp:44]: Somewhat. Maps would still load, but events may not be configured correctly (assuming that we have maps in vcmi format with daily events)

File: lib/CSoundBase.h (hunk 2 above)
Comment (IvanSavenko) [reply lib/CSoundBase.h:262]: Yes. 
These values refer to names of files inside H3 sound archive.
To be honest we can remove most of these values - they are kind of deprecated, with most of them now migrated to config.

File: lib/CGameInfoCallback.cpp (hunk 1 above)
Comment (Alexander-Wilms) [reply lib/CGameInfoCallback.cpp:789]: Fixed in https://github.com/vcmi/vcmi/pull/4195

**Pull request merged**